src/lib/card.js

```diff
@@ -0,0 +1,101 @@
+export default function createCard(card) {
+  const cardSection = document.createElement('section')
+  cardSection.classList.add('question-card')
+
+  const bookmark = document.createElement('i')
+  bookmark.classList.add(
+    'far',
+    'fa-bookmark',
+    'question-card__bookmark',
+    'question-card__bookmark--toggled'
+  )
+  cardSection.append(bookmark)
+
+  if (card.isBookmarked) {
+    bookmark.classList.remove('far')
+    bookmark.classList.add('fas')
+  } else {
+    bookmark.classList.remove('fas')
+    bookmark.classList.add('far')
+  }
+
+  bookmark.addEventListener('click', () => {
+    bookmark.classList.toggle('far')
+    bookmark.classList.toggle('fas')
+  })
+
+  const headlineWrapper = document.createElement('div')
```

Comment:
not sure how to name it better. I now called it headWrapper, but since it is just a div that sits around the headline (for flex purposes) I have no better descriptive name for it.

Reply:
Maybe then`flexWrapper` for a variable name?